src/pages/Shop/Components/ItemDetail/Components/ItemReviewList/Components/ItemReview.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import './ItemReview.scss'
+
+class ItemReview extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isHide: true,
+      nestedComment: '',
+      commentList: []
+    }
+  }
+
+  showMore = () => {
+    this.setState({
+      isHide: !this.state.isHide
+    })
+  }
+
+  inputHandler = (e) => {
+    this.setState({
+      nestedComment: e.target.value
+    })
+  }
+
+  addComment = (evt) => {
+    evt.preventDefault();
+    const {commentList, nestedComment} = this.state
+    this.setState({
+      commentList: [...commentList, 
+      {
+        id: commentList.lenght +1,
+        user: 'minddon',
+        comment: nestedComment
+      },
+    ],
+      nestedComment: ''
+    })
+  }
+
+  render() {
+    const {id, option, comment, imgUrl, userId, postedDate} = this.props;
+    const {isHide, commentList, nestedComment} = this.state;
+
+    return(
+      <li className={isHide ? "ItemReview" : "ItemReview opened"}>
+        <div className="reviewContent">
+          <div onClick={() => this.showMore()}>
```

Comment:
아래와 같이 작성하시는게 더욱 깔끔할 것 같네요!
```suggestion
          <div onClick={this.showMore}>
```